tests/containers/suma_containers.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test SUMA container images:
+#   The scope of the testing is minimal and covers pulling and running 
+#   container images from the internal registry.suse.de
+#   
+# The container images tested are:
+#   - proxy-httpd
+#   - proxy-salt-broker
+#   - proxy-squid
+#   - proxy-ssh
+#   - proxy-tftpd
+#
+# The testing of proxy-charts is limited to pulling the tar from the registry to
+# and inspect the content to ensure Charts is available. 
+# Install or deployment is not covered in this scenario.
+#   
+# In depth testing of all the images  is already done by the SUMA QE team
+#
+# Maintainer: Maurizio Galli <maurizio.galli@suse.com>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    my $image = get_required_var('CONTAINER_IMAGE_TO_TEST');
+
+    if (check_var('CONTAINER_SUMA', 'image')) {
+        script_retry("podman pull $image", timeout => 300, delay => 60, retry => 3);
+        assert_script_run("podman run --rm $image cat /etc/os-release");
```

Comment:
Improvement suggestion: Use [validate_script_output](https://open.qa/api/testapi/#_validate_script_output) instead of `assert_script_run` and check, that `/etc/os-release` contains what it's supposed to be containing.

That's a suggestion, feel free to ignore it.

Reply:
The purpose of the test is only to ensure the container is running and we can run commands inside it. If the container isn't running then `cat` command will fail making the test fail. 

I never considered the output of os-release to be that important in this scenario and I could have picked a different command. The reasoning for picking os-release is that it is present in every product and it's just:
```
NAME="SLES"
VERSION="15-SP4"
VERSION_ID="15.4"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP4"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp4"
DOCUMENTATION_URL="https://documentation.suse.com/"
```

This does not mean we shoulnd't follow your recommendation if it's important

Reply:
I see, then perhaps executing `true` instead of `cat` might be better comprehensible? But the current solution works, so if you like it, just ignore this comment :slightly_smiling_face:

Reply:
`cat /etc/os-release` suggests that someone would like to check if the values there make sense.

`true` suggests to me, this is just a smoke test.

Reply:
so just replacing `cat /etc/os-release` with `true` is enough?

Reply:
If it's just a smoke test, yes - unless you think that `cat /etc/os-release` is better suited!

Reply:
yea it's essentially a smoke test for now. The SUMA QE team may eventually decide to improve upon these tests but for now at maintenance we just want something simple to automate the container release.

Reply:
I replaced `cat /etc/os-release` with `true`